.pre-commit-config.yaml

```diff
@@ -3,7 +3,6 @@
 # See https://github.com/pre-commit/pre-commit-hooks/blob/master/.pre-commit-config.yaml for an example with more hooks
 
 ci:
-  autofix_prs: false
   autoupdate_commit_msg: "ENH: Autoupdate hooks [pre-commit.ci]"
```

Comment:
if this is the commit that happens in the PR I don't think it should have ENH in front - maybe better something like '[pre-commit.ci] pre-commit fixes'?

Reply:
It's actually the default PR title for the autoupdates, not for commits. We can change the PR prefix to `MNT` after #224.